src/main/resources/hudson/plugins/sidebar_link/Messages.properties

```diff
@@ -1,4 +1,3 @@
-Uploaded=Uploaded image file; use {0} in Link Icon field above.
-NoFile=No file uploaded.
-DupName=File with this name already exists in userContent.
-Back=Back
+uploaded=Uploaded image file; use {0} in Link Icon field above.
```

Comment:
do we need to change it to lowercase ?

Reply:
checkstyle complained that the methods used in java start with a capital letter

Reply:
it did not complain so far so I don't see reason why it starts now

Reply:
It was spotbugs see https://ci.jenkins.io/job/Plugins/job/sidebar-link-plugin/job/PR-58/1/spotbugs/
Probably because I changed the Messages.properties

Reply:
can you revert those changes - I don't see value of them, just key renaming and loosing translations which were deleted

Reply:
Done.
What might be done is move the ja translations from the deleted `startUpload_ja.properties`to the `config_ja.properties`. The texts are identical except for one where just the traliing `:` is removed.